stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -1,5 +1,9 @@
 package com.stripe.android;
 
+import android.util.Log;
+
+import java.util.concurrent.Executor;
```

Comment:
Nope, that's fine. Warnings aren't great, so I'd rather we lock everything down to a specific version.